ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.5.2.yaml

```
@@ -0,0 +1,223 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
+  name: knative-v0.3
```

```suggestion
  name: knative-v0.5.2
```

Sorry the script doesn't catch this on its own.